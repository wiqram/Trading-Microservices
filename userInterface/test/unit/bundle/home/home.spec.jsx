import React from 'react';
import { create} from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../../../src/store/index';
import Home from '../../../../src/app';

jest.mock('../../../../src/bundle/product/home', () => () => 'ProductHome');
jest.mock('../../../../src/bundle/transaction/home', () => () => 'TransactionHome');

describe('Home Snapshot Test', () => {

    it('renders correctly with all props given', () => {
        const wrapper = create(<Provider store={store}><Home /></Provider>)
            .toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});