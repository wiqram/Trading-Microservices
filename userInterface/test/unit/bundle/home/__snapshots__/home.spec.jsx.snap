// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Snapshot Test renders correctly with all props given 1`] = `"ProductHome"`;
